datumaro/components/dataset.py

```diff
@@ -1003,6 +1005,12 @@ def detect(path: str, *,
         if 1 < len(matches):
             raise MultipleFormatsMatchError(matches)
 
+    @classmethod
+    def check_media_type(cls, source: IDataset):
```

Comment:
```suggestion
    def _check_media_type(cls, source: IDataset):
```
No reasons to make it public.

Reply:
Fixed